File: Lab3/app/src/main/java/com/example/pamo/lab3/quiz/Question.java
package com.example.pamo.lab3.quiz;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Question implements Serializable {

    private final static long serialVersionUID = 8810718296371414610L;

    private String name;
    private List<Answer> answers = new ArrayList<>();

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Answer> getAnswers() {
        return answers;
    }

    public void setAnswers(List<Answer> answers) {
        this.answers = answers;
    }
}

File: Lab4/app/src/main/java/com/example/pamo/lab4/quiz/Question.kt
package com.example.pamo.lab4.quiz

import java.io.Serializable
import java.util.*

class Question : Serializable {
    var name: String? = null
    var answers: MutableList<Answer> = ArrayList()

    companion object {
        private const val serialVersionUID = 8810718296371414610L
    }
}